src/pages/favorites/favorites.component.ts

```diff
@@ -164,6 +167,14 @@ export class FavoritesComponent {
   }
   deleteTrip(trip): void {
     _.remove(this.trips, {name: trip.name});
-    this.tripSvc.deleteTrip(trip);
+    this.faveTripSvc.deleteTrip(trip);
+  }
+  removeRoute(route): void {
+    _.remove(this.routes, {name: route.name});
```

Comment:
FYI the structure of all the Avail objects (like Route, RouteDetails, Stop, StopDeparture, etc) is currently best found on [the Swagger doc](http://bustracker.pvta.com/InfoPoint/swagger/ui/index).